src/main/java/io/airbyte/testingtool/scenario/instance/AirbyteConnection.java

```diff
@@ -31,4 +41,46 @@ public ConnectionIdRequestBody getConnectionRequestBody() {
     return connectionIdRequestBody;
   }
 
+  public void setNormalization(boolean destinationSupportNormalization) throws ApiException {
+    ConnectionIdRequestBody connectionIdRequestBody = new ConnectionIdRequestBody();
+    connectionIdRequestBody.setConnectionId(getConnectionId());
+    ConnectionRead connectionRead = airbyteInstance.getAirbyteApi().getConnectionApi().getConnection(connectionIdRequestBody);
+
+    if (destinationSupportNormalization) {
```

Comment:
this validation doesn't make sense here. you should get isNormalizationOn as input, but not destination

Reply:
fixed it. Thanks for noticing this